### ISSUE TITLE
Capture Region: If you drag the region above the viewport, the capture doesn't happen

### ISSUE THREAD

**Issue by GoogleCodeExporter** [cf/chrome-screen-capture#77]
```
What steps will reproduce the problem?

1. Initiate a "Capture Region" (the region box with grab handles appears)

2. Drag the upper-left ("NW corner") drag handle up to the top of the 
viewport/page and then up beyond the top of the page. You won't actually be 
able to see the cursor anymore, but it will be above the browser viewport. In 
other words, you are dragging it up to where the links bar/address bar are.

3. Click OK

What is the expected output? What do you see instead?
EXPECTED: The selected region I selected and captured shows in the new tab, 
ready for editing.
ACTUAL: Region selection disappears but nothing else happens.


What version of the product are you using? On what operating system?
Version 3.0.5 Win 7 64

Please provide any additional information below.
Basically, if the top of the region is above the viewport (< 0 pixels 
positionally), the capture seems to fail. This only happens on the top. The 
left, right, and bottom are bounded by the browser window when the region is 
being resized, which is what I would expect to happen, and probably what should 
be happening at the top as well. The page I was testing it on (this bug 
submission page) did not have any scrollbars, though I don't think that makes a 
difference for reproducing the bug.
```

Original issue reported on code.google.com by `content%...@gtempaccount.com` on 11 Jan 2011 at 8:10
